crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -116,6 +116,17 @@ def repocheck
       end
     end
 
+    def ceph_healthy?
+      ceph_node = NodeObject.find("roles:ceph-mon AND ceph_config_environment:*").first
+      return true if ceph_node.nil?
+      ssh_retval = ceph_node.run_ssh_cmd("ceph health 2>&1")
```

Comment:
Yep. From @tserong "it'll return 0 even if there's a mon down and it's in HEALTH_WARN"